deals/deals.go

```diff
@@ -183,15 +190,15 @@ func pushNewChanges(ctx context.Context, api API, currState map[cid.Cid]*api.Dea
 				Size:          dinfo.Size,
 				PricePerEpoch: dinfo.PricePerEpoch.Uint64(),
 				Duration:      dinfo.Duration,
-				DealID:        dinfo.DealID,
+				DealID:        uint64(dinfo.DealID),
```

Comment:
`DealID` didn't exist in `testnet/3`, actually I was the one that included it in `master` to use here.
Did the same PR yesterday but in this branch: https://github.com/filecoin-project/lotus/pull/1452

Reply:
Forgot to mention, `DealID` was necessary for a reason... it is needed in other API for getting the Epoch at which the deal became active.